pkg/contract-manager/models.go

```diff
@@ -0,0 +1,33 @@
+package contractmanager
+
+// Selector represents the criteria for selecting Flavours.
+type Selector struct {
+	FlavourType      string `json:"type,omitempty"`
+	CPU              int    `json:"cpu,omitempty"`
+	RAM              int    `json:"ram,omitempty"`
+	EphemeralStorage int    `json:"ephemeral-storage,omitempty"`
```

Comment:
i.e. `Arch` for the cpu architecture? this is an example of standard k8s information

Reply:
Here we go!